react/features/etherpad/middleware.js

```diff
@@ -40,7 +40,7 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
 StateListenerRegistry.register(
     state => getCurrentConference(state),
     (conference, { dispatch, getState }, previousConference) => {
-        if (conference) {
+        if (conference && !getState()['features/base/conference'].authRequired) {
```

Comment:
Why is this needed? `getCurrentConference` will return the new conference on each willJoin?

Reply:
Are you not refactoring this one?

Reply:
This is the only thing missing I think!

Reply:
Oh I missed that comment, working on it.